exampleCourse/questions/fibonacciEditorWithR/tests/tests.R

```diff
@@ -0,0 +1,47 @@
+# Unit tests with `testthat` use the behavior-driven development (BDD) format
+# instead of the traditional `test_that` format. This allows for two "contexts"
+# to be created.
+
+# In particular, we make the following assumptions about unit tests:
+
+# 1. `describe()` function indicates a single problem steps worth.
+# 2. `it()` function describes the specification being assessed.
+#
+# Avoid using multiple comparisons within the same `describe()`
+
+# Sample test
+describe("1", {                               # Specify points per check
+    it("Does `fib()` exist?", {               # Description of test
+        source("../student/student_answer.R") # Load student submission
+        expect_equal(class(fib), "function")  # Perform test
+    })
+})
+
+describe("1", {
+    it("Check fib(0)", {
+        source("../student/student_answer.R")
+        expect_equal(fib(0), 0)
```

Comment:
Given that we didn’t actually specify what should happen for `fib(0)` in the question, can we remove this test? I tend to think we should have fixed checks for `fib(1) = 1`, `fib(2) = 1`, `fib(3) = 2`, `fib(4) = 3`, and then the randomized tests listed below.